ampersand-view.js

```diff
@@ -297,6 +289,22 @@ assign(View.prototype, {
         this.on('change', action, this);
     },
 
+    // Parser the declarative subview definition.
+    // Overload this method to create your own declarative subview style
+    _parseSubviewOpts: function (subview, name) {
```

Comment:
Are you waiting for me to change it or perhaps we could just commit it and then I can clean that up with some additional documentation?